### PR TITLE
Returned VLM chat sample in CI.

### PR DESCRIPTION
**Pull request by popovaan**
Returned VLM chat sample in CI.

### CODE REVIEW

File: .github/workflows/causal_lm_cpp.yml [hunk 1]
```diff
@@ -716,6 +716,43 @@ jobs:
...
+          python -m pip install ./thirdparty/openvino_tokenizers/[transformers] --pre --extra-index-url https://storage.openvinotoolkit.org/simple/wheels/nightly
+          python ./samples/cpp/visual_language_chat/export_MiniCPM-V-2_6.py ./miniCPM-V-2_6/
+          wget https://github.com/openvinotoolkit/openvino_notebooks/assets/29454499/d5fbbd1a-d484-415c-88cb-9986625b7b11
+          
+      - name: Run chat sample
+        run: |
+          source ./ov/setupvars.sh
+          export PYTHONPATH=./build/:$PYTHONPATH
+          printf 'What is on the image?\nWhat is special on the image?\n' > ./input.txt
+          timeout 120s python ./samples/python/visual_language_chat/visual_language_chat.py ./miniCPM-V-2_6/ d5fbbd1a-d484-415c-88cb-9986625b7b11 < input.txt > ./pred.txt
```
Comment (Wovchena) [inline .github/workflows/causal_lm_cpp.yml:754]: Why don't you run python in the same job?

File: .github/workflows/causal_lm_cpp.yml [hunk 2]
```diff
@@ -716,6 +716,13 @@ jobs:
           && timeout 120s ./build/samples/cpp/visual_language_chat/visual_language_chat ./miniCPM-V-2_6/ cat.jpg
           <<< $'What is on the image?\nWhat is special on the image?'
 
+      - name: Run python chat sample
```
Comment (Wovchena) [inline .github/workflows/causal_lm_cpp.yml:719]: You need to update `cmake --build ./build/ --config Release --target visual_language_chat -j` line by adding `py_generate_pipeline` target

File: samples/python/visual_language_chat/visual_language_chat.py [hunk 3]
```diff
@@ -54,12 +54,20 @@ def main():
     config.max_new_tokens = 100
 
     pipe.start_chat()
+    try:
+        prompt = input('question:\n')
+    except EOFError:
```
Comment (ilya-lavrenov) [inline samples/python/visual_language_chat/visual_language_chat.py:59]: do we need to have try / except? 
It moves focus from GenAI to exception handling within python

**Review by ilya-lavrenov — APPROVED** [review]

File: samples/python/visual_language_chat/visual_language_chat.py (hunk 3 above)
Comment (popovaan) [reply samples/python/visual_language_chat/visual_language_chat.py]: Agree. On first iteration it is probably redundant.

**Pull request merged**